### DISCUSSION TITLE
GDA2020 support

### DISCUSSION THREAD

**Discussion by Eskapp** [pyproj4/pyproj#836] [Q&A]
This stems from an issue that I currently have: I want to convert a pair of latitude/longitude in UTM in the GDA2020 datum.
From my experiments I am thinking that this is not supported. Maybe it is just the fact that I don't use the library properly so let me explain the way I have been trying to realize this tramsformation:

Let's say thatI want to convert the (latitude, longitude) pair: (-27.45745, 153.05715) to UTM in the GDA2020 datum. The UTM zone is 56J.

```
from pyproj import Proj
proj_utm = Proj('proj=utm +zone=16J, +south +ellps=GRS80 +datum=GDA2020 +units=m +no_defs')
```

returns the following error:

`pyproj.exceptions.CRSError: Invalid projection: +proj=utm +zone=56J, +south +ellps=GRS80 +datum=GDA2020 +units=m +no_defs +type=crs: (Internal Proj Error: proj_create: Error -9 (unknown elliptical parameter name))`

The error message is a bit inaccurate as the ellipsoid `GRS80` is actually known if I use the datum `NAD83` (for North American pairs of latitude/longitude).

Is the GDA2020 datum defined in Pyproj? If not, is there a way to do this conversion using the library?
I stumbled across posts to convert from GDA94 to GDA2020 but I am struggling to get the GDA1994 to work in pyproj.

Thanks!

**Comment by snowman2** [comment]
https://pyproj4.github.io/pyproj/stable/build_crs.html
```python
from pyproj.crs import GeographicCRS, ProjectedCRS
from pyproj.crs.coordinate_operation import UTMConversion

proj_crs = ProjectedCRS(
    conversion=UTMConversion("16", "S"), geodetic_crs=GeographicCRS(datum="GDA2020")
)
```
This produces:
```
<Projected CRS: {"$schema": "https://proj.org/schemas/v0.2/projjso ...>
Name: undefined
Axis Info [cartesian]:
- E[east]: Easting (metre)
- N[north]: Northing (metre)
Area of Use:
- undefined
Coordinate Operation:
- name: UTM zone 16S
- method: Transverse Mercator
Datum: Geocentric Datum of Australia 2020
- Ellipsoid: GRS 1980
- Prime Meridian: Greenwich
```
